### ISSUE TITLE
change module layout and groupId 

### ISSUE THREAD

**Issue by jangalinski** [camunda/camunda-bpm-assert#110]
With the "newer" extensions, we introduced a imho cleaner setup:

* the extension is a multi-module project with two main sub-modules "extension" and "examples", where examples are configured to be ignored by the deployment so they do not end up on central
* the groupId should be: org.camunda.bpm.extension.assert, so not all artifacts of all extensions land in the same directory
* (optional) all the root poms have the groupId "o.c.b.e.assert.project" so they are clearly separated from the core extension modules

I would love to see this new structure for the 2.0 release (which is  a perfect cut for a new groupId) and volunteer to create the PR.

**Comment by tmetzke** [comment]
Hi Jan,

thanks for proposing this!

I adjusted the structure of the project and incorporated the "examples" module (plus its exclusion from deployment) and the ".project" addition to the root pom's `groupId`.
Also, the other artifacts now have a more specific `groupId` including ".assert".

For now, I did not introduce an "extension" sub-module, since this is strictly-speaking no extension anymore.

Let me know what you think of the new structure.
Thanks again!

Best,
Tobias
